### ISSUE TITLE
Casting columns to boolean doesn't work

### ISSUE THREAD

**Issue by himanshu-pro** [pganalyze/pg_query#138]
Command: `PgQuery.parse("Select x::boolean").deparse`

Expected: `SELECT "x"::boolean`

Actual: `SELECT FALSE`

**Comment by himanshu-pro** [comment]
This is because expected behaviour for `PgQuery.parse("Select 't'::boolean").deparse` is `SELECT TRUE` but `deparse_typecast` is not checking if `node['arg']` is `A_CONST` or not..

**Comment by lfittl** [comment]
Fixed in 2.0:

```
irb(main):011:0> PgQuery.parse("Select x::boolean").deparse
=> "SELECT x::boolean"
irb(main):012:0> PgQuery.parse("Select false").deparse
=> "SELECT false"
```
